ui/src/me/components/GettingStarted.tsx

```diff
@@ -23,8 +23,7 @@ class GettingStarted extends PureComponent<WithRouterProps> {
             <GradientBorder />
             <CollectorGraphic />
             <h3 className="getting-started--title">
-              Configure a<br />
-              Data Collector
+              Load your data
```

Comment:
Super nitpicky, but all the cards in the getting started page have text on 2 lines, so I am wondering if there's a way to make them all 1 line or to make this 2 lines for consistency